scripts/restore

```diff
@@ -75,6 +75,7 @@ ynh_script_progression --message="Restoring user rights..."
 
 # Restore permissions on app files
 chown -R $app: $final_path
+chmod -R 775 "$final_path/storage"
```

Comment:
```suggestion
chmod 775 "$final_path/storage"
```

c.f. comments on other apps, not a big fan of running a `chmod` that recursively set every regular file as executable, if the intention is in fact to just define perms on the parent dir

Also is it intentional to `chmod 775` instead of `chmod 770` ?